options/locale/locale_en-US.ini

```
@@ -403,7 +403,7 @@ openid_connect_title = Connect to an existing account
 openid_connect_desc = The chosen OpenID URI is unknown. Associate it with a new account here.
 openid_register_title = Create new account
 openid_register_desc = The chosen OpenID URI is unknown. Associate it with a new account here.
-openid_signin_desc = Enter your OpenID URI. For example: https://anne.me, bob.openid.org.cn or gnusocial.net/carry.
+openid_signin_desc = Enter your OpenID URI. For example: https://bob.openid.org.cn or gnusocial.net/carry.
```

I think we can use Wikipedia's example if we'd like to improve it:  https://en.wikipedia.org/wiki/OpenID

```suggestion
openid_signin_desc = Enter your OpenID URI. For example: alice.openid.example.org or https://openid.example.org/alice.
```

---

Good idea. I think I'm done here for the rest of this decade (or until I hear about more users "doing things wrong").